source/process/sg_grammar-spelling-mechanics.rst

```diff
@@ -0,0 +1,148 @@
+
+Preferred
+  The cows ran from wolves, coyotes, and mosquitoes.
+
+Avoid
+  The cows ran from wolves, coyotes and mosquitoes.
+
+Tone
+----
+
+Use a direct, impartial tone. Most readers of the documentation are looking for answers and solutions to their problems; they are not looking for entertainment.
+
+Preferred
+  If your password is rejected, check to make sure that Caps Lock is off, and then carefully type it in again. 
+
+Avoid
+  Failed sign in? No problem! Simply enter the correct password and we'll let you in right away.
+
+.. _capital:
+
+Capitalization
+--------------
+
+Use title case for page titles and sentence case for section titles.
+
+Title case
+  Grammar, Spelling, and Mechanics
+
+Sentence case
+  Language and spelling
+
+Voice
+-----
+
+Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it. Use passive voice only when you want to emphasize the action more than the subject.
+
+Preferred
+  The system opens the *Status* pane.
+
+Avoid
+  The *Status* pane is opened by the system.
+
+Person
+------
+
+Use the second person and avoid the first person.
+
+Preferred
+  You can view the status in the *Status* pane.
+
+Avoid
+  We'll view the status in the *Status* pane.
+
+Numbers
+-------
+
+Spell out numbers when they are the first word in a sentence, otherwise use numeric digits.
+
+Use commas to make long numbers easier to read.
+
+Preferred
+  Three cows ran for 6 kilometers when they saw 2,300,097 mosquitoes chasing them.
+
+Avoid
+  3 cows ran for six kilometers when they saw 2300097 mosquitoes chasing them.
+
+Text highlighting
+-----------------
+
+Use highlighting of text to visually set off words and phrases that are important to readers. Content that should be highlighted includes file names, UI controls, and window titles. The following table has a comprehensive list with examples. 
+
+==============  ==================  =======================
+Text            Highlight           Example
+==============  ==================  =======================
+file name       ``monospace``       ``config.py``
+directory name  ``monospace``       ``/opt/mattermost``
+inline code     ``monospace``       ``fmt.Printf("2 times %d = %d\n", x, y )``
+code samples    ``monospace``       See :ref:`syntax-highlight` for an example.
+screen output   ``monospace``       See :ref:`literal-blocks` for an example.
+menu selection  **bold**            "Click **File > Save**."
+UI selection    **bold**            "Click **Next**."
+field names     **bold**            "Enter the font in the **Display Font** field."
+commands        ``monospace``       "At the command line, type ``sudo apt-get install nginx``."
+citations       *italic*            "Read the book *Clean Code* by Robert Martin."
+window titles   *italic*            "The *Account Settings* window opens."
```

Comment:
Few tweaks:

1) Please also help add a guideline for
 - keyboard buttons: all capitals with no spaces and no monospace highlighting. Example: CTRL+U
 - placeholder fields: inside square brackets, no other highlighting. Example: [team name]
 - key strokes in monospace: Example: `10`

2) Capitalize the fields under "Text" column (maybe add this part of guidelines for tables)

3) Maybe group `commands` text formatting with the other monospace options. Perhaps below code samples.

4) Propose replacing "menu selection" and "UI selection" with "Clickable Control", which covers both